pkg/virtctl/imageupload/imageupload.go

```diff
@@ -439,6 +440,8 @@ func ConstructUploadProxyPathAsync(uploadProxyURL, token string, insecure bool)
 }
 
 func (c *command) uploadData(token string, file *os.File) error {
+	const maxRetries = 15
```

Comment:
Should we do what we did with export and make this a proper arg with a default of 2?

Reply:
ACK